benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -227,6 +227,14 @@ msgctxt "image alt text"
 msgid "core.icons.bankcard"
 msgstr "Bank card icon with contactless symbol"
 
+#: benefits/eligibility/forms.py:23
+msgid "eligibility.form.continue"
+msgstr "Continue"
```

Comment:
We used to use `msgid "eligibility.pages.index.button"` for this, can we just update that text instead of creating a new entry?